compiler-rt/lib/builtins/cpu_model/x86.c

```diff
@@ -36,14 +36,14 @@ enum VendorSignatures {
   SIG_AMD = 0x68747541,   // Auth
 };
 
-enum ProcessorVendors {
+enum ProcessorVendors : unsigned int {
```

Comment:
Is this ISO-compliant C? In [C17 standard](https://www.open-std.org/jtc1/sc22/wg14/www/docs/n2310.pdf) 6.7.2.2 _enum-specifier_ has no _enum-base_ which I am afraid that is only a C++11 feature.

Reply:
Ok. _enum-type-specifier_ is a C23 feature and only supported in gcc-13+. Is this `: unsigned int` important here and can we get rid of it in favor of compatibility with older compilers?